src/nomad_simulations/schema_packages/properties/__init__.py

```diff
@@ -18,7 +18,33 @@
 
 from .band_gap import ElectronicBandGap
 from .band_structure import ElectronicBandStructure, ElectronicEigenvalues
-from .energies import ChemicalPotential, FermiLevel
+from .energies import (
+    FermiLevel,
+    BaseEnergy,
```

Comment:
do we need to import BaseEnergy here?

Reply:
I use this within the h5md parser for identifying all energy classes with properties with:
```python
        self.energy_classes = {m[1].__name__.replace('Energy', '').lower(): m[1]
                          for m in inspect.getmembers(energy_module, inspect.isclass)
                          if 'Energy' in m[1].__name__}
```
So, yes in principle it can be useful. But if you prefer not to have it so accessible I can do a more detailed import?